app/market/facilitator_test.go

```diff
@@ -16,13 +16,13 @@ import (
 )
 
 func TestFacilitator(t *testing.T) {
-	state_ := testutil.NewState(t, nil)
-	account, key := testutil.CreateAccount(t, state_)
+	commitState, _ := testutil.NewState(t, nil)
+	account, key := testutil.CreateAccount(t, commitState)
 
 	nonce := uint64(10)
 
 	account.Nonce = nonce
-	state_.Account().Save(account)
+	commitState.Account().Save(account)
```

Comment:
Error return value of `(github.com/ovrclk/akash/state.AccountAdapter).Save` is not checked